crypto/LPdir_win.c

```diff
@@ -104,6 +104,9 @@ const char *LP_find_file(LP_DIR_CTX **ctx, const char *directory)
              * the ending NUL.
              */
             if ((len_0 - 1) > MAX_PATH * sizeof(TCHAR)) {
+                if (extdirbuf != NULL) {
```

Comment:
you don't need the if check; `free(NULL);` works fine.

Reply:
Agreed.

You'll also need to add "CLA: trivial" on a line by itself in the commit message (use git commit --amend) or submit a signed ICLA.

Reply:
Done. Check please.

Reply:
Thanks.